plugins/javadoc/src/main/kotlin/javadoc/renderer/JavadocContentToHtmlTranslator.kt

```diff
@@ -35,6 +35,19 @@ internal class JavadocContentToHtmlTranslator(
         }
     }.joinToString("<br>", """<span class="code">""", "</span>") { it }
 
+    private fun sanitize(content: String): String {
+        val replacements = listOf(">" to "&gt;", "<" to "&lt;")
```

Comment:
We do have a helper in core that does exactly that I believe

Reply:
yup, removed